back_end/tweeter/lib/tweeter_web/controllers/page_controller.ex

```diff
@@ -0,0 +1,9 @@
+defmodule TweeterWeb.PageController do
```

Comment:
We dont need this file or this controller for our project , so we could remove it